docs/reference-manual/native-image/Bundles.md

```diff
@@ -186,24 +207,83 @@ You successfully rebuilt the application from the bundle with debug info enabled
 
 The full option help of `--bundle-apply` shows a more advanced use case that will be discussed [later](#combining---bundle-create-and---bundle-apply) in detail:
 ```
---bundle-apply=some-bundle.nib
+--bundle-apply=some-bundle.nib[,dry-run][,container[=<container-tool>][,dockerfile=<Dockerfile>]]
                       an image will be built from the given bundle file with the exact same
                       arguments and files that have been passed to native-image originally
                       to create the bundle. Note that if an extra --bundle-create gets passed
                       after --bundle-apply, a new bundle will be written based on the given
-                      bundle args plus any additional arguments that haven been passed
+                      bundle args plus any additional arguments that have been passed
                       afterwards. For example:
                       > native-image --bundle-apply=app.nib --bundle-create=app_dbg.nib -g
                       creates a new bundle app_dbg.nib based on the given app.nib bundle.
                       Both bundles are the same except the new one also uses the -g option.
 ```
 
+
+### Building in a Container
+
+Another addition to the `--bundle-create` and `--bundle-apply` options, as mentioned above, is to perform image building inside a container image.
```

Comment:
Put every sentence into its own line (as this avoids merge conflicts on future edits).

Reply:
Addressed in commit https://github.com/oracle/graal/pull/6496/commits/83db0c17b7812eac5e15f70d23d47ae0fe14cd44